TYPO3.Neos/Classes/TYPO3/Neos/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -382,7 +382,11 @@ protected function translateAndConvertHelpMessage(array &$configuration, $idPref
     protected function addTargetAttribute($htmlString)
     {
         $document = new \DOMDocument();
-        $document->loadHTML($htmlString);
+
+        // Force correct unicode handling
+        $document->loadHTML('<?xml encoding="UTF-8">' . $htmlString);
+        $document->encoding = 'UTF-8';
```

Comment:
@bwaidelich I tried removing it, but it does no harm anyways, and it kind of makes sense to have it there. Doesn't affect the markdown output in any way of course.
@kdambekalns yes, the other way would be to add targets with regexp, but  that's even more scary.